src/components/PersonLink/index.tsx

```diff
@@ -0,0 +1 @@
+export * from './PerosnLink';
```

Comment:
Error: The import path './PerosnLink' contains a typo. It should be './PersonLink' to match the corrected filename and ensure proper imports.